sdk/python/lib/pulumi/runtime/resource.py

```diff
@@ -363,7 +365,7 @@ def do_resolve(value: Any, perform_apply: bool, exn: Optional[Exception]):
                 supportsPartialValues=True,
             )
 
-            from ..resource import create_urn
+            from ..resource import create_urn # pylint: disable=import-outside-toplevel
```

Comment:
why did this get triggered now?

Reply:
Looks like pylint got bumped when I installed mypy: https://github.com/pulumi/pulumi/pull/3758/files#diff-5d560596e48dd61e776a2e6b034ee710R181-R187